### ISSUE TITLE
[Supporting platform - Make code - Share project]: After setting the viewport to 320*256px, Controls present on the 'Share dialog' are truncated

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5471]
### User Experience:
Low vision users who depend on zooming functionality for better readability will be impacted if the content present on the page is not visible or truncated after setting the viewport to 320*256px. They will not be able to access the share the project controls to share the project.

### Note:
User credentials should NOT be included in the bug.

### Prerequisite:
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of "Share Project" dialog.
7. Observe whether controls present in the 'Share project' dialog is truncated or not.

### Actual Result:
Issue: After setting the viewport to 320*256px, Controls present on the 'Share dialog' are truncated

**Observation:**
Controls like 'X', 'Embed code' etc... which are present on the dialog are truncated and unable to access with both keyboard and mouse.

### Expected Result:
After setting the viewport to 320*256px, Controls present on the 'Share dialog' should be visible clearly without truncation.

**Note:**
The same issue exists throughout the application.

### MAS Reference:
[MAS 1.4.10 – Reflow](https://aka.ms/MAS1.4.10)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:

NA

### UIA/DOM details available:

NA

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.
<img width="493" alt="MAS1 4 10_After setting the viewport to 320x256pixels, Controls present on the 'Share dialog' are truncated" src="https://github.com/microsoft/pxt-microbit/assets/158465724/f7ba0037-6e18-43f0-a05e-e82c624d17e9">
<img width="960" alt="Normal mode_Share project dialog" src="https://github.com/microsoft/pxt-microbit/assets/158465724/a91ecb47-bc0a-4f0d-b052-384ac3609bc6">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#A11yWCAG2.1;#MAS1.4.10;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Jan2024;

**Comment by vroopea** [comment]
As checked the issue on URL: https://makecode.microbit.org/. The issue is still repro and severity is fine. #Regressed:04-08-24;InternalReviewed-Apr2024
